test/functional/home_controller_test.rb

```diff
@@ -19,6 +19,16 @@ class HomeControllerTest < ActionController::TestCase
     end
   end
 
+  context "without redis" do
+    should "still render page" do
```

Comment:
I'd consider making the naming reflect something closer to the resiliency matrix section/dependency as it makes it easier to evolve. Example:

<img src="https://cloud.githubusercontent.com/assets/97400/8446801/7d9a4952-1f7d-11e5-9e90-7d624a044d2f.png" width=500 />

Making this something like:

```ruby
context "with redis down" do
  should "render front page" do
    # ..
  end
end
```

Super minor and cosmetic, but the mental model of the matrix really helps ironing out all the corners in my experience.